.github/workflows/ci.yml

```diff
@@ -16,106 +16,106 @@ jobs:
-         else
-           cmake -S "${{github.workspace}}" -B "${{github.workspace}}/build" -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_QFR_TESTS=ON
-         fi  
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name:  Configure CMake
+        shell: bash
+        run:   |
+               if [ "$RUNNER_OS" == "Windows" ]; then
+                 cmake -S "${{github.workspace}}" -B "${{github.workspace}}/build" -DCMAKE_BUILD_TYPE=$BUILD_TYPE -T "ClangCl" -DBUILD_QFR_TESTS=ON
+               else
+                 cmake -S "${{github.workspace}}" -B "${{github.workspace}}/build" -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_QFR_TESTS=ON
+               fi
+
+      - name:  Build
+        shell: bash
+        run:   cmake --build "${{github.workspace}}/build" --config $BUILD_TYPE --target qfr_test
 
-    - name: Build
-      shell: bash
-      run: cmake --build "${{github.workspace}}/build" --config $BUILD_TYPE --target qfr_test
+      - name:              Test
+        working-directory: ${{github.workspace}}/build/test
+        shell:             bash
+        run:               |
+                           if [ "$RUNNER_OS" == "Windows" ]; then
+                             cd $BUILD_TYPE
+                             ./qfr_test.exe
+                           else
+                             ./qfr_test
+                           fi
 
-    - name: Test
-      working-directory: ${{github.workspace}}/build/test
-      shell: bash
-      run: |
-        if [ "$RUNNER_OS" == "Windows" ]; then
-          cd $BUILD_TYPE
-          ./qfr_test.exe
-        else
-          ./qfr_test
-        fi
-    
   coverage:
     needs: test
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-      with:
-        submodules: recursive
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.8'
-    - name: Configure CMake
-      shell: bash
-      run: cmake -S "${{github.workspace}}" -B "${{github.workspace}}/build" -DCMAKE_BUILD_TYPE=Debug -DCOVERAGE=1 -DBUILD_QFR_TESTS=ON
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name:  Configure CMake
+        shell: bash
+        run:   cmake -S "${{github.workspace}}" -B "${{github.workspace}}/build" -DCMAKE_BUILD_TYPE=Release -DCOVERAGE=1 -DBUILD_QFR_TESTS=ON
 
-    - name: Build
-      shell: bash
-      run: cmake --build "${{github.workspace}}/build" --config Debug --target qfr_test
+      - name:  Build
+        shell: bash
+        run:   cmake --build "${{github.workspace}}/build" --config Release --target qfr_test
```

Comment:
Shouldn't the coverage run in the debug build?

Reply:
Yeah, definitely. This has been changed since the build times in debug mode were horrendous. But since this has been resolved, it should be safe to go back to debug mode.